interfaces/functions.def

```diff
@@ -463,6 +463,16 @@ igraph_get_shortest_paths_dijkstra:
               weights ON graph
         IGNORE: RR, RC, RNamespace
 
+igraph_get_shortest_paths_bellman:
```

Comment:
The function that you've added is called `igraph_get_shortest_paths_bellman_ford`